plugins/essif-lab_contactform7/wordpress-plugin.php

```diff
@@ -17,34 +9,38 @@
  * Text Domain: essif-lab_contactform7
  */
 
-// If this file is called directly, abort.
-if ( ! defined( 'WPINC' ) ) {
+if ( ! defined( 'WPINC' ) || !defined('ABSPATH') ) {
 	die;
 }
 
-define( 'ESSIF_LAB_CONTACTFORM7_VERSION', '1.0.0' );
+$classAutoloader = __DIR__.'/vendor/autoload.php';
+if (file_exists($classAutoloader)) {
+    require_once($classAutoloader);
+}
+
+use TNO\ContactForm7\Application\Essif_Lab_contactform7_Activator;
+use TNO\ContactForm7\Application\Essif_Lab_contactform7_Deactivator;
+use TNO\ContactForm7\Application\Essif_Lab_contactform7;
 
 function activate_essif_lab_contactform7() {
-	require_once plugin_dir_path( __FILE__ ) . 'Application/class-essif-lab_contactform7-activator.php';
+	require_once plugin_dir_path( __FILE__ ) . 'src/Application/activator.php';
     $activator = new Essif_Lab_contactform7_Activator();
```

Comment:
Probeer de plugin vanuit een "main" class te instantiëren zoals we ook in de hoofd plugin doen. Stel als doel voor dit bestand om de hoofd class te instantiëren en daarbij al het instelwerk te doen (dus implementaties voor interfaces definiëren en koppelen).